docs/ref/settings.txt

```diff
@@ -2075,7 +2075,9 @@ Uses of the key shouldn't assume that it's text or bytes. Every use should go
 through :func:`~django.utils.encoding.force_text` or
 :func:`~django.utils.encoding.force_bytes` to convert it to the desired type.
 
-Django will refuse to start if :setting:`SECRET_KEY` is not set.
+Django will refuse to start if :setting:`SECRET_KEY` is set to an empty value.
```

Comment:
This might not be accurate. runserver raises an error but running the tutorial project with gunicorn doesn't.